pydax/typing.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2020 IBM Corp. All Rights Reserved.
```

Comment:
```suggestion
# Copyright 2021 IBM Corp. All Rights Reserved.
```

Reply:
Can we update the copyright year separately for all the files at once? I added a note to this issue: https://github.com/CODAIT/pydax/issues/163

Reply:
Yeah it's fine